DEVELOPMENT.md

```diff
@@ -15,8 +15,24 @@ Individual scripts can be iteratively developed, debugged, and tested using the
 
 First, build a baseline of the image to include the packages needed by existing or new scripts:
 
-```shell script
-docker build -t mc-dev .
+PowerShell:
+```powershell
+$env:IMAGE_TO_TEST="mc-dev"
```

Comment:
Thanks. I always have to google for that syntax and then re-discover how simple it is.